internal/ast/compiler/prometheusdataquery.go

```diff
@@ -0,0 +1,45 @@
+package compiler
+
+import (
+	"github.com/grafana/cog/internal/ast"
+)
+
+var _ Pass = (*PrometheusDataquery)(nil)
+
+// PrometheusDataquery rewrites the definition of the prometheus.Dataquery type and adds a few missing fields.
+// Note: this pass is meant to be removed once the schema is up-to-date.
+type PrometheusDataquery struct {
+}
+
+func (pass *PrometheusDataquery) Process(schemas []*ast.Schema) ([]*ast.Schema, error) {
+	for i, schema := range schemas {
+		schemas[i] = pass.processSchema(schema)
+	}
+
+	return schemas, nil
+}
+
+func (pass *PrometheusDataquery) processSchema(schema *ast.Schema) *ast.Schema {
+	for i, object := range schema.Objects {
+		if schema.Package == prometheusPackage && object.Name == prometheusDataqueryObject {
+			schema.Objects[i] = pass.processDataquery(object)
+			continue
```

Comment:
That continue doesn't do anything, right?

Reply:
Yeah, just a habit :)